tests/integration/test_manifest_generation.py

```diff
@@ -318,10 +346,17 @@ def test_single_manifest_generation_google_sheet(
         assert columns["key_datetime"] is not None
         assert columns["entityId"] is not None
 
-        assert (
-            sheet1[f"{columns['Filename']}2"].value
-            == "Manifest generation - Manual test - generate an existing manifest/test dataset/test dataset 1/sample A.txt"
-        )
+        if local_flask_instance:
```

Comment:
This was a little odd that there is a difference here between the local instance and the deployed instance, anyone have ideas here?

Reply:
I haven't looked closely, the dev instance is not the most up to date release yet - so there may be differences.

Reply:
@BryanFauble Tom is right. The file path feature has not yet been deployed (so there are differences). After we deploy, this ` if local_flask_instance:` can be removed.

Reply:
When is the dev instance being deployed? Shouldn't this already be deployed if the ask is that we are running through the test plan?

Isn't everything a moot point for testing against the dev instance of schematic if we aren't running the version of schematic in dev we want to test against?

Reply:
Thanks for the questions @BryanFauble . We can deploy to AWS dev any time (there's no technical blockers). But based on FAIR data team's current release process, we go through test plans before we release to PYPI. Usually after PYPI release, we would release to aws dev environment. However, because of the bug related to nested data structure Schematic v24.10.1 was deleted in PYPI and we never deploy to AWS dev. 

But you are right! I agree that we could expedite deploying to AWS dev since that will make testing changes on AWS sooner.